addons/zylann.hterrain/native/quad_tree_lod_generic.gd

```diff
@@ -70,18 +62,49 @@ func set_split_scale(p_split_scale: float):
 
 	# Split scale must be greater than a threshold,
 	# otherwise lods will decimate too fast and it will look messy
-	if p_split_scale < MIN:
-		p_split_scale = MIN
-	if p_split_scale > MAX:
-		p_split_scale = MAX
-
-	_split_scale = float(p_split_scale)
+	_split_scale = clamp(p_split_scale, MIN, MAX)
 
 
 func get_split_scale() -> float:
 	return _split_scale
 
 
+func clear():
```

Comment:
Why did you move these down?

Reply:
I matched the CPP version and organized the functions that call each other together. Clear, join all recursively, create_sizes, was one group. Update, _update, another. All of the smaller, individual functions at the top. Debug funcs at the bottom. 

Do you want me to revert to the original order? Or just move a few functions?

After you specify that, I'll rebuild and push all the changes.

Reply:
Having them match is not enforced. But the main issue is that you mixed "formatting" with logic changes, it makes it harder to review code. Preferably, formatting should be in separate commits.

Also, the C++ class is not meeting the standards I use (see ImageUtils, and [this](https://voxel-tools.readthedocs.io/en/latest/module_development/#code-guidelines)), however as I said earlier I'm no longer working with GDNative as there will probably be a big rewrite (not just a port) of the plugin with GDExtension in the future, so it's not a big deal